src/rf/apps/core/decorators.py

```diff
@@ -43,7 +56,10 @@ def wrapper(request, *args, **kwargs):
                 raise MethodNotAllowed()
 
             if request.method == 'PUT':
-                request.PUT = QueryDict(request.body)
+                request.PUT = PatchedQueryDict(request.PUT.urlencode())
+
+            if request.method == 'POST':
+                request.POST = PatchedQueryDict(request.POST.urlencode())
```

Comment:
Was there a reason behind using `request.POST.urlencode()` instead of `request.body`?